File: GARI_Umbria_2026-08-03/GiasNG/src/app/Model/anagrafiche/SottogruppoStalla.ts
import { Razza, TipoGruppo_Zoo, TipologiaCapoAnimale } from "app/Service/api.service";
import { SottogruppoStallaLight } from "./SottogruppoStallaLight";
import { IntervalloTemporale } from "./IntervalloTemporale";
import { Specie } from "../metaschema/utilizzi/Specie";

export class SottogruppoStalla extends SottogruppoStallaLight {

    tipo: TipoGruppo_Zoo;

    validita: IntervalloTemporale;

    area: number;

    specie: Specie;

    razza: Razza;

    stato: TipologiaCapoAnimale;

}

File: GARI_Umbria_2026-08-03/DotNet/AgronicaCoreStd/AgronicaCoreModelsSTD/anagrafiche/SottogruppoStalla.cs
using System;
using System.Collections.Generic;
using System.Text;
using AgronicaCoreModelsSTD.metaschema;
using AgronicaCoreModelsSTD.metaschema.utilizzi;

namespace AgronicaCoreModelsSTD.anagrafiche
{
    public class SottogruppoStalla : SottogruppoStallaLight
    {
        #region Properties
        //public string nome { get; set; }
        //public int codice { get; set; }
        //public Fabbricato.PK stallaPK { get; set; }

        /// <summary>
        /// 
        /// </summary>
        public TipoGruppo_Zoo tipo {get; set;}

        /// <summary>
        /// 
        /// </summary>
        public IntervalloTemporale validita { get; set; }

        /// <summary>
        /// Area del recinto in mq
        /// </summary>
        public double area { get; set; }


        /// <summary>
        /// Specie animale presente
        /// </summary>
        public Specie specie { get; set; }

        /// <summary>
        /// Razza animale presente
        /// </summary>
        public Razza razza { get; set; }

        /// <summary>
        /// 
        /// </summary>
        public TipologiaCapoAnimale stato { get; set; }
        #endregion


    }
}
